basics/persisting_state_of_components.md

```diff
@@ -79,29 +79,17 @@ State class should have a `equals` method, but if it is not implemented, state o
 
 In order to specify where exactly the persisted values will be stored, you need to add a `@State` annotation to the `PersistentStateComponent` class. It has the following fields:
 
-* `name` (required) - specifies the name of the state (name of the root tag in XML)
-* One or more of `@com.intellij.openapi.components.Storage` annotations (required) - specify the storage locations for `.ipr` and directory-based projects
-* `reloadable` (optional) - if set to false, complete project reload is required when the XML file is changed externally and the state has changed.
+* `name` (required) — specifies the name of the state (name of the root tag in XML).
+* `storages` — one or more of `@com.intellij.openapi.components.Storage` annotations to specify the storage locations. Optional for project level values — standard project file will be used in this case.
```

Comment:
What is 'standard project file'? I suppose it is *.ipr file for file-based scheme, but what about the default directory-based scheme?